Source/WebInspectorUI/Localizations/en.lproj/localizedStrings.js

```diff
@@ -2000,3 +2000,33 @@ localizedStrings["unsupported version"] = "unsupported version";
 localizedStrings["value"] = "value";
 /* Placeholder text in an editable field for the value of a HTTP header */
 localizedStrings["value @ Local Override Popover New Headers Data Grid Item"] = "value";
+localizedStrings["Resolution"] = "Resolution";
+localizedStrings["Video Format"] = "Video Format";
+localizedStrings["Audio Format"] = "Audio Format";
+localizedStrings["Media Details"] = "Media Details";
+localizedStrings["Video Codec"] = "Video Codec";
+localizedStrings["Color"] = "Color";
+localizedStrings["Video Details"] = "Video Details";
+localizedStrings["Audio Codec"] = "Audio Codec";
+localizedStrings["Sample Rate"] = "Sample Rate";
+localizedStrings["Channels"] = "Channels";
+localizedStrings["Audio Details"] = "Audio Details";
+localizedStrings["Transfer Function"] = "Transfer Function";
+localizedStrings["Color Primaries"] = "Color Primaries";
+localizedStrings["Matrix Coefficients"] = "Matrix Coefficients";
```

Comment:
Many, if not all, of these strings really need unique keys and descriptions since they are very domain-specific, and thus that context is critical for localizers. e.g. `WI.UIString("Matrix Coefficients", "Matrix Coefficients (Media Details Sidebar Panel)", "Title label for the matrix coefficients of a video/audio element")`